src/sdk/pynni/nni/nas/pytorch/base_trainer.py

```diff
@@ -4,13 +4,13 @@
 class BaseTrainer(ABC):
 
     @abstractmethod
-    def train(self):
+    def train(self, validate=True):
```

Comment:
does not make sense, please remove

Reply:
You mean `validate=True` doesn't make sense? Why?